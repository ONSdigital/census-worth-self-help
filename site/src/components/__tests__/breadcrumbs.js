import React from "react"
import renderer from "react-test-renderer"

import Breadcrumbs from "../breadcrumbs"

describe("Breadcrumbs", () => {
  it("renders correctly", () => {

  	let breadcrumbs = [{title:'root', link:'root'}, {title:'parent', link:'parent'}]
  	let peers = [{title:'peer1', link:'peer1'}, {title:'peer2', link:'peer2'}, {title:'self', link:'self'}]

    const tree = renderer
      .create(<Breadcrumbs breadcrumbs={breadcrumbs} peers={peers}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})