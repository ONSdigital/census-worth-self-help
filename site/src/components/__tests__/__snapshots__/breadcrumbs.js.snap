// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "ypl6yy",
      "next": undefined,
      "styles": "
        border-bottom: 1px rgb(229, 229, 234) solid;
        padding: 15px 0px;
      ",
    }
  }
>
  <div
    className="Section-heading-Style"
    css={
      Object {
        "map": undefined,
        "name": "136nmlb",
        "next": undefined,
        "styles": "
          
    padding: 5px 15px;
  
        ",
      }
    }
  >
    EXPLORE CONTENT
  </div>
  <div
    css={
      Object {
        "map": undefined,
        "name": "136nmlb",
        "next": undefined,
        "styles": "
          
    padding: 5px 15px;
  
        ",
      }
    }
  >
    <select
      css={
        Object {
          "map": undefined,
          "name": "8a47ri",
          "next": undefined,
          "styles": "
            background-color: rgb(255, 255, 255);
            border-bottom: rgb(48, 48, 48) 1px solid;
            width: 100%;
            padding: 5px;
          ",
        }
      }
      onChange={[Function]}
    >
      <option
        value="root"
      >
        root
      </option>
      <option
        value="parent"
      >
        parent
      </option>
      <option
        value="peer1"
      >
        peer1
      </option>
      <option
        value="peer2"
      >
        peer2
      </option>
      <option
        value="self"
      >
        self
      </option>
    </select>
  </div>
</div>
`;
